pages/index.js

```diff
@@ -2,7 +2,42 @@ import { atom, useAtom } from "jotai";
 import Link from "next/link";
 import { useRouter } from "next/router";
 
-export const globalReviews = atom({ good: 0, neutral: 0, bad: 0 });
+export const globalReviews = atom({
+  good: {
+    count: 0,
+
+    aspects: {
+      hospitality: 0,
+      cleanness: 0,
+      foodQuailty: 0,
+      foodQuantity: 0,
+      ambience: 0,
+      friendliness: 0,
+    },
```

Comment:
Isn't there a setter function in jotai?
You can look into the Zustand workshop it's a similar concept about setters and getters
It's easier to set/get those aspects later on. 
It's just the complexity increases with more with so many aspects.
You have to add 18 setters to modify each aspect and 18 getters to retrieve them